lib/components/narrative/realtime-annotation.js

```diff
@@ -63,15 +63,15 @@ export default class RealtimeAnnotation extends Component {
 
     if (componentClass === 'popover') {
       return <OverlayTrigger
-        trigger='click'
-        placement='bottom'
-        // container={this}
-        // containerPadding={40}
         overlay={
-          <Popover style={{maxWidth: '300px'}} id='popover-positioned-bottom'>
+          <Popover id='popover-positioned-bottom' style={{maxWidth: '300px'}}>
             {innerContent}
           </Popover>
-        }>
+        }
+        placement='bottom'
+        // container={this}
+        // containerPadding={40}
```

Comment:
Are these comments still needed?

Reply:
They seem from a different era, I removed them in https://github.com/opentripplanner/otp-react-redux/pull/390/commits/11773ca2755c074d3ca83db0e40a92031c737b97.